hotham-simulator/src/state.rs

```diff
@@ -20,8 +20,15 @@ use std::{
     time::Instant,
 };
 
-use crate::{inputs::Inputs, simulator::NUM_VIEWS, space_state::SpaceState};
-// use crate::simulator::spa
+use crate::{
+    action_state::ActionState, inputs::Inputs, simulator::NUM_VIEWS, space_state::SpaceState,
+};
+
+static X_INPUT: &str = "/user/hand/left/input/x/click";
+static Y_INPUT: &str = "/user/hand/left/input/y/click";
+static B_INPUT: &str = "/user/hand/right/input/b/click";
+static A_INPUT: &str = "/user/hand/right/input/a/click";
```

Comment:
I think we should stick with alphabetic order as much as possible to keep things consistent.

Reply:
Ah that's reasonable!